recipes/diligent-core/all/conanfile.py

```diff
@@ -169,6 +173,11 @@ def package_info(self):
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Graphics", "GraphicsTools", "interface"))
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Graphics", "HLSL2GLSLConverterLib", "interface"))
 
+        if self.settings.os == 'Windows':
+            self.cpp_info.defines.append("FILE2STRING_PATH=${CONAN_BIN_DIRS_DILIGENT-CORE}/File2String.exe")
+        else:
+            self.cpp_info.defines.append("FILE2STRING_PATH=${CONAN_BIN_DIRS_DILIGENT-CORE}/File2String")
```

Comment:
```suggestion
        if self.settings.os == 'Windows':
            self.cpp_info.defines.append("FILE2STRING_PATH=%s" % os.path.join(self.cpp_info.rootpath, "bin", "File2String.exe")
        else:
            self.cpp_info.defines.append("FILE2STRING_PATH=%s" % os.path.join(self.cpp_info.rootpath, "bin", "File2String")
```
As it is currently, it can not work if the cmake generator is not used.